server/public/model/feature_flags.go

```diff
@@ -69,6 +72,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.TestBoolFeature = false
 	f.EnableRemoteClusterService = false
 	f.EnableSharedChannelsDMs = false
+	f.EnableSharedChannelsMemberSync = true
```

Comment:
set to ""true" for now , for testing in the cloud, for some reason the cloud slash command feature flag param didn't work.